libs/components/router/src/lib/modules/href/href.directive.ts

```diff
@@ -49,8 +53,12 @@ export class SkyHrefDirective {
    */
   @Input()
   public set skyHrefElse(value: 'hide' | 'unlink') {
```

Comment:
Same here.

Reply:
Fixed. I don't love this one as only 'hide' matter and so to avoid a breaking change I had to treat `undefined` like `unlink` but it is what it is.